src/main/java/net/io/kino/model/Showroom.java

```diff
@@ -0,0 +1,54 @@
+package net.io.kino.model;
+
+import com.fasterxml.jackson.annotation.JsonIdentityInfo;
+import com.fasterxml.jackson.annotation.JsonIdentityReference;
+import com.fasterxml.jackson.annotation.ObjectIdGenerators;
+
+import javax.persistence.*;
+import java.util.List;
+
+
+@Entity
+@JsonIdentityInfo(generator = ObjectIdGenerators.PropertyGenerator.class, property = "id")
+public class Showroom {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    public Showroom(String name, int numberOfSeats) {
+        this.name = name;
+        this.noOfSeats = numberOfSeats;
+
+        for (int i = 0; i <numberOfSeats ; i++) {
```

Comment:
Ostatnio uzgadniliśmy na labach, że showroom będzie miał określone wymiary tzn ilość kolumn i wierszy foteli w sali. Tworzenie rekordu do każdego fotela w każdej sali chyba mijałoby się z celem skoro bilet ma mieć określone położenie w sali

Reply:
Czyli Showroom będzie wiedział jakie ma wymiary, a zajęte miejsca będziemy obliczać na podstawie aktywnych  Order i Ticketów w nich.

Reply:
Zrozumiałem to tak, że mam w ogóle nie robić pola na miejsca w Showroomie, bo będzie to obsłużone w ticketach. Tak?

Reply:
Tak. Showroom ma mieć tylko wymiary sali (prostokąt)

Reply:
No to zrobione. Zerknijcie teraz i ew. kolejne komentarze